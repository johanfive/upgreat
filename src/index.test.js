const cli = require('./')

it('should export cli', () => {
  expect(cli).toMatchSnapshot()
})
