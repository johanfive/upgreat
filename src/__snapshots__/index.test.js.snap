// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should export cli 1`] = `
Object {
  "Command": [Function],
  "Main": [Function],
  "default": [Function],
  "flags": Object {
    "boolean": [Function],
    "build": [Function],
    "enum": [Function],
    "help": [Function],
    "integer": [Function],
    "option": [Function],
    "string": [Function],
    "version": [Function],
  },
  "run": [Function],
}
`;
